client/webpack.config.prod.js

```diff
@@ -1,15 +1,15 @@
-const path = require("path");
+const path = require('path');
 const webpack = require('webpack');
 const BundleTracker = require('webpack-bundle-tracker');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 const ExtractAppCSS = new ExtractTextPlugin({
   filename: 'css/app.css',
-  allChunks: true
+  allChunks: true,
```

Comment:
Any reason why we have lots of trailing commas?

Just curious, as I thought it's only typescript project that applies.

Reply:
It doesn't apply to just typescript, it's a general eslint rule, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas